### ISSUE TITLE
Adding raw SVG markup via .svg() requires xmlns (in Pure/Standalone SVG context)

### ISSUE THREAD

**Issue by simonwidjaja** [svgdotjs/svg.js#723]
In a pure SVG file using SVG.js I need to add nodes from markup via .svg(rawString).

This is NOT working in pure SVG (in HTML wrapper it works just fine):
```
      this.stage.svg(
        '<rect fill="#BADA55" width="200" height="200" x="400" y="400"/>'
      )
```

After some hours of try/error I copied some markup from chrome's dev tools and it suddenly worked. Isolating the issue it seems that I need to manually add xmlns="http://www.w3.org/2000/svg" to every single tag in my markup to make .svg(rawString) work from within the pure SVG file...

This works:
```
      this.stage.svg(
        '<rect xmlns="http://www.w3.org/2000/svg" fill="#BADA55" width="200" height="200" x="400" y="400"/>'
      )
```

Is that by design / expected behaviour or is it a bug?
Used in a HTML/JS/SVG context it works just fine without xmlns... So all my scripts from that context do NOT work in pure SVG.

**Comment by Fuzzyma** [comment]
Please checkout the 3.0 branch and check if this happens there too.

**Comment by simonwidjaja** [comment]
sorry didn't find time sooner to test this...
I have loaded 3.0. now but getting lots of errors.
e.g.
+ SVG('stage') has to be SVG('#stage') now
+ .select("#HOTSPOT_TPL").first() // first() is not a function
etc.
Is there already some kind of documentation/changelog for 3.0.0?

**Comment by Fuzzyma** [comment]
Iam sorry. These changes where applied _after_ i asked you to checkout 3.0.
Try bc9bfb6025e826b0ee6c827f1a356995d7f05d4c instead. You can use it with `git checkout bc9bfb6025e826b0ee6c827f1a356995d7f05d4c`

**Comment by simonwidjaja** [comment]
Will checkout that specific commit in a sec.
Quick question regarding changelog:
I cannot find anything for changes to ".first()".
only: 

> changed childNodes to children (same for firstChild, lastChild, ...) (#710)

This line is a little confusing. Is it now .firstChild() instead .first() or is it .firstChildren()?

**Comment by Fuzzyma** [comment]
That's an internal change. Not related to any method of svg.js. SVG.Set was removed and therefore every method which returned a set now returns an array. SVG.Set might be back soon in a better version but that's a bunch to think through. 

You can access the first element of a select with simple array notation

**Comment by simonwidjaja** [comment]
perfect already did that. working.
also had to change 

var image = this.scene.image(data.image.src, data.image.width, data.image.height)
to
var image = this.scene.image(data.image.src).size(data.image.width, data.image.height)

and everything is up and running again!
So no need to checkout the older commit.
AND: also no need manually setting the namespace for each and every element in template strings. So this issue is fixed with current 3.0 branch

Is there any document or info you can share regarding the rough roadmap?

**Comment by Fuzzyma** [comment]
Cool - nice to hear that it works without problems in 3.0.

I roughly follow the [3.0 project](https://github.com/svgdotjs/svg.js/projects/1) when working on the 3.0 branch. There is no real roadmap beside this.
